_episodes/08-CI.md

```diff
@@ -440,7 +460,7 @@ Back to the Codecov `molecool` report page, the sunburst plot gives a hierarchic
 <center><img src="../fig/08_CI/codecov_screenshot.png"></center>
 <br />
 
-[//]: # TODO: Check if the following feature work in GitHub Action, this paragraph is the legacy from Travis CI
+[//]: # TODO: Check if the following feature work in GitHub Actions, this paragraph is the legacy from Travis CI
```

Comment:
I'm previewing locally, and it seems like this line is hidden when you preview the page. Is that correct?